llvm/utils/update_mca_test_checks.py

```diff
@@ -65,18 +65,7 @@ def _parse_args():
         help="the binary to use to generate the test case " "(default: llvm-mca)",
     )
     parser.add_argument("tests", metavar="<test-path>", nargs="+")
-    args = common.parse_commandline_args(parser)
-
-    _configure_warnings(args)
-
-    if not args.llvm_mca_binary:
```

Comment:
I think we should also preserve the checks from line 72 to 76

Reply:
Updated, but moved to main since `parse_commandline_args` is called in main.